lib/main.go

```diff
@@ -214,7 +215,33 @@ func (obj *Main) Run() error {
 		return fmt.Errorf("hostname cannot be empty")
 	}
 
+	// Get current user running mgmt
+	user, _ := user.Current()
```

Comment:
You always have to handle errors!

Reply:
bah my bad. I did that to test, and forgot to check how you were handling errors. I'll fix that.